### PR TITLE
Set up theme as submodule

### PR DESCRIPTION
**Pull request by helenmiller16**
This will hopefully simplify the setup a bit and make it easier to update the theme, since we won't have to track all the theme files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,14 @@ To get this repo locally clone it down, then run `hugo serve -w` (the `-w` is fo
...
 
 With a pull request, you'll get a preview of the site, click on `Details` to get a link to the preview site. If it fails you'll also see that.
+
+## Theme
+
+The Hugo Boostrapious theme is hosted on [github](https://github.com/devcows/hugo-universal-theme), and managed here as a git [submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules). To update the theme: 
+
+```
+cd themes/hugo-universal-theme
+git pull
```
Comment (k8hertweck) [inline README.md]: I had to specify `git pull origin master`

Comment (k8hertweck) [reply README.md]: Also, you can't run `hugo serve -w` inside `hugo-universal-theme/`; it would be worth explaining this clearly in the instructions

**Comment by k8hertweck** [comment]
I think this is great! Apart from the few comments I have about improving the README, I'm wondering if we should add `themes` to `.gitignore`? I haven't made conversions to submodules before, though, so this may not be necessary?

I suspect this may also take care of some security things I've been dragging my feet over resolving, (e.g. #67).

**Comment by helenmiller16** [comment]
I updated the README with some more explicit instructions for cloning and updating the theme. I'm new to git submodules, so figuring it out. It's what Hugo docs suggested, and seems like a great idea to me! You shouldn't have to specify `theme` in `.gitignore`. Git is smart about all that. It just tracks the submodule remote url, its commit hash, and the directory it's supposed to go in.

**Pull request merged**